testing/coreruleset/coreruleset_test.go

```diff
@@ -280,8 +315,30 @@ func crsWAF(t testing.TB) coraza.WAF {
 	if err != nil {
 		t.Fatal(err)
 	}
+	customTestingConfig := `
```

Comment:
I added this so the benchmark runs the same rules as FTW, I'm not sure if there is a practical difference though